src/mf/mf-uvc.cpp

```diff
@@ -213,7 +213,7 @@ namespace librealsense
                                 auto& stream = owner->_streams[dwStreamIndex];
                                 std::lock_guard<std::mutex> lock(owner->_streams_mutex);
                                 auto profile = stream.profile;
-                                frame_object f{ current_length, metadata_size, byte_buffer, metadata, monotonic_to_realtime(llTimestamp/10000.f) };
+                                frame_object f{ current_length, metadata_size, byte_buffer, metadata, monotonic_to_realtime(llTimestamp * 0.0001) };
```

Comment:
Can you explain?

Reply:
the floating type caused loss of data. mult instead of div is only for number of mathematical operations optimization

Reply:
Got it thanks